packages/flex-plugin-scripts/README.md

```diff
@@ -10,6 +10,62 @@ Scripts to run, build, and deploy Flex plugin.
 
 This repo will automatically be installed as part of creating a new plugin.
 
+## Scripts
+
+### Release
+
+```bash
+npm run release {major, minor, patch, custom, overwrite} {--public}
+```
+
+This script will build, and then deploy your plugin bundle to Twilio Assets. Please note that your accountSid and authToken are required to use this script. Your plugins will be uploaded to Twilio Assets using the following format:
+
+```text
+https://domain.twil.io/plugins/${PluginName}/${Version}/bundle.js
+```
+
+Where `PluginName` is the `name` from `package.json`, and `Version` is the `version` from `package.json`. For example:
+
+```text
+https://domain.twil.io/plugins/dialpad/1.0.0/bundle.js
+https://domain.twil.io/plugins/dialpad/1.1.0/bundle.js
+https://domain.twil.io/plugins/dialpad/1.1.1/bundle.js`
+```
+
+You may choose between `major, minor, patch, custom, overwrite` options:
+
+* **major** A major SemVer release version, e.g. going from 1.2.3 to 2.0.0. Example: `npm run release major` 
+* **minor** A minor SemVer release version, e.g. going from 1.2.3 to 1.3.0. Example: `npm run release minor`
+* **patch** A patch SemVer release version, e.g. going from 1.2.3 to 1.2.4. Example: `npm run release patch`
+* **custom** A custom release version if you are not using SemVer versioning. Provide the version as the next argument. Example `npm run release custom v1.2-someTag`
```

Comment:
missing colon after Example

i actually think it might be easier to read if you put the examples in an indented new line